src/charts/donut/Readme.md

```diff
@@ -38,6 +38,15 @@
 ```
 
 
+### With loading state
+```js
+
+    <Donut
+        data={null}
+        shouldShowLoadingState={true}
```

Comment:
Let's talk about this API tomorrow. Maybe it is redundant and we can simplify it.

Reply:
Definitely open to discussing improvements here! FWIW I really wanted to make a withLoadingState HOC but didn't go that route because the loading state lives in the chart (e.g. `Donut.js`) layer. If we can think of a way to get around this I think that be preferable.

Reply:
As commented in person, let's go with the 'shouldShowLoadingState' option, to be more explicit